src/main/java/dev/andrylat/vomelianchuk/finances/cardnumbervalidator/dialog/CardNumberValidatorDialog.java

```diff
@@ -0,0 +1,29 @@
+package dev.andrylat.vomelianchuk.finances.cardnumbervalidator.dialog;
+
+import dev.andrylat.vomelianchuk.finances.common.Dialog;
+import dev.andrylat.vomelianchuk.finances.cardnumbervalidator.consoleio.CardNumberValidatorConsoleIO;
+import dev.andrylat.vomelianchuk.finances.cardnumbervalidator.exceptions.CardValidationException;
+import dev.andrylat.vomelianchuk.finances.cardnumbervalidator.exceptions.WrongPaymentSystemException;
+import dev.andrylat.vomelianchuk.finances.cardnumbervalidator.validation.CardNumberValidator;
+
+public class CardNumberValidatorDialog implements Dialog {
+    static final String DESCRIPTION_ACTION = "Card Number Validation";
+
+    @Override
+    public void start() {
+        var consoleIO = new CardNumberValidatorConsoleIO(System.in, System.out);
```

Comment:
Maybe better to inject values of input \ output streams instead of propagating them here System.in, System.out

Reply:
Fixed here: d7b1b80